std/bigint.d

```diff
@@ -1964,3 +1964,735 @@ BigInt powmod(BigInt base, BigInt exponent, BigInt modulus) pure nothrow
     BigInt result = powmod(base, exponent, modulus);
     assert(result == 359079);
 }
+
+/// for isProbablePrime
+private
+{
+    bool millerRabinPrimeTest(const BigInt n, const BigInt[] bases, const size_t confidence)
+    {
+        BigInt[] test_bases;
+
+        if (bases.length)
+        {
+            test_bases = bases.dup;
+        }
+        else
+        {
+            import std.random : Mt19937, unpredictableSeed, uniform;
+            Mt19937 rnd;
+            rnd.seed(unpredictableSeed);
+
+            test_bases = new BigInt[confidence];
+            for (int i = 0; i < confidence; i++)
+            {
+                test_bases[i] = uniform!("[]", uint, uint, typeof(rnd))(uint.min, uint.max, rnd) % (n - 2) + 2;
```

Comment:
I think the intention here is to generate a ` BigInt` whose minimum value is 2 and whose maximum value is `n - 2`, but this cannot produce values greater than `uint.max -2` (4,294,967,293) and this branch is only taken when `n` > 10,000,000,000,000,000.

Reply:
@n8sh, Thank you very much.
There's a problem here. I've fixed it.